site/content/docs/desktop/addons/report-generation/_index.md

```diff
@@ -113,6 +113,16 @@ The pattern that will be used to generate the report name.
 
 The directory from which templates will be loaded. You should not typically need to change unless you are designing new reports, in which case it can be useful to change to point to your local source code reports directory. The "reports" folder in your ZAP home directory is used by default.
 
+### Field Descriptions
+
+#### riskdesc
+
+riskdesc - Is a combination identifier, showing Risk followed by Confidence (in brackets). For example `High (Medium)` would indicate a High risk issue identified with Medium confidence.
```

Comment:
followed is broken across lines?

Reply:
Not sure what the question is. The whole sentence is in a single line.

Reply:
Seems to be a display artifact in my tablet

![image](https://github.com/user-attachments/assets/21d7f045-ad7f-4199-a539-c3430c41b44a)